lib/vkloader.c

```diff
@@ -749,46 +749,62 @@ linearTilingPadCallback(int miplevel, int face,
  * limited number of formats and features. Generally @c VK_IMAGE_TILING_OPTIMAL
  * is preferred. The latter requires a staging buffer so will use more memory
  * during loading.
+ * 
+ * If a pointer to a set of suballocator callbacks is provided, they
+ * will be used instead of manual allocation of VkDeviceMemory. A 64 bit uint
+ * that references the suballocated page(s) is returned on memory procurement 
+ * and saved in the @c allocationId field of the structure pointed to by @A vkTexture.
```

Comment:
s/@A/@a/. An error in my request. Sorry.

Reply:
Fixed!